tutorials/Qubit/Qubit.ipynb

```diff
@@ -0,0 +1,309 @@
+    "\n",
+    "Second, they are **orthogonal** to each other:\n",
+    "\n",
+    "$$\\langle \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} , \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} \\rangle =\n",
+    "\\langle \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix} , \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix} \\rangle = 0$$\n",
+    "\n",
+    "> As a reminder, $\\langle V , W \\rangle$ is the [inner product](../LinearAlgebra/LinearAlgebra.ipynb#Inner-Product) of $V$ and $W$.\n",
+    "\n",
+    "This means that these vectors form an **orthonormal basis**. To be more specific, this is the **computational basis**, also known as the **canonical basis**.\n",
+    "\n",
+    "> There exist other orthonormal bases, such as, for example, the **Hadamard basis**:\n",
+    ">\n",
+    "> $$\\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{bmatrix} \\text{and} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ -\\frac{1}{\\sqrt{2}} \\end{bmatrix}$$\n",
+    ">\n",
+    "> $$\\begin{bmatrix} \\alpha \\\\ \\beta \\end{bmatrix} =\n",
+    "\\frac{\\alpha + \\beta}{\\sqrt{2}} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ \\frac{1}{\\sqrt{2}} \\end{bmatrix} +\n",
+    "\\frac{\\alpha - \\beta}{\\sqrt{2}} \\begin{bmatrix} \\frac{1}{\\sqrt{2}} \\\\ -\\frac{1}{\\sqrt{2}} \\end{bmatrix}$$\n",
+    ">\n",
+    "> The Hadamard basis is widely used in quantum computing, for example in the [BB84 quantum key distribution protocol](https://en.wikipedia.org/wiki/BB84)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dirac Notation\n",
+    "\n",
+    "Completely writing out each vector when doing quantum calculations already takes up a lot of space, and this will get even worse when we introduce quantum gates and multi-qubit systems. Luckily, there is a way around this using **Dirac notation**. In Dirac notation, a vector is represented by a symbol called a **ket**. For example, a qubit in state $0$ is represented by the ket $|0\\rangle$, and a qubit in state $1$ is represented by the ket $|1\\rangle$:\n",
+    "\n",
+    "<table>\n",
+    "    <col width=150>\n",
+    "    <col width=150>\n",
+    "    <tr>\n",
+    "        <td style=\"text-align:center; border:1px solid\">$|0\\rangle = \\begin{bmatrix} 1 \\\\ 0 \\end{bmatrix}$</td>\n",
+    "        <td style=\"text-align:center; border:1px solid\">$|1\\rangle = \\begin{bmatrix} 0 \\\\ 1 \\end{bmatrix}$</td>\n",
+    "    </tr>\n",
+    "</table>\n",
+    "\n",
+    "Because these two kets represent basis states, they can be used to represent any other state:\n",
+    "\n",
+    "$$\\begin{bmatrix} \\alpha \\\\ \\beta \\end{bmatrix} = \\alpha|0\\rangle + \\beta|1\\rangle$$\n",
+    "\n",
+    "A symbol other than $0$ or $1$ within the ket can be used to represent arbitrary vectors, similar to how variables are used in algebra, although a few symbols have a generally accepted use, such as:\n",
```

Comment:
I'd split this line into several separate thoughts:
```suggestion
    "Any symbol other than $0$ or $1$ within the ket can be used to represent arbitrary vectors, similar to how variables are used in algebra: $|\\psi\\rangle = \\alpha|0\\rangle + \\beta|1\\rangle$.\n",
```
And the next line would be "Several ket symbols have a generally accepted use:"